### PR TITLE
Tests added to just-cd

### PR DESCRIPTION
**Pull request by manasdadheech**

### CODE REVIEW

File: just_cd/deploy_app/views.py [hunk 1]
```diff
@@ -94,33 +93,36 @@ def process_status(
...
+        file_path = os.path.join(settings.BASE_DIR,
+                                 reponame, 'just_config.yaml')
         with open(file_path, 'r') as stream:
             try:
                 return yaml.load(stream)
             except yaml.YAMLError as exc:
-                print(exc)
+                return exc
+    else:
+        return ''
```
Comment (manasdadheech) [inline just_cd/deploy_app/views.py]: Return `None` instead. You do not need to return anything

File: just_cd/deploy_app/tests/test_open_config_true.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    def test_open_config_file_correct(self):
+        result = open_just_config('test_repo')
+        print(result)
+        assert ('Deploy' in result)
+
+    def tearDown(self):
+        path = os.path.join(settings.BASE_DIR, 'test_repo')
+        # os.remove(os.path.join(settings.BASE_DIR,
+        #           'test_repo', 'just_config.yaml'))
+        # os.rmdir(path)
```
Comment (manasdadheech) [inline just_cd/deploy_app/tests/test_open_config_true.py]: The comments can be removed

File: just_cd/deploy_app/pytest.ini [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[pytest]
+DJANGO_SETTINGS_MODULE=just_cd.settings.test
```
Comment (kracekumar) [inline just_cd/deploy_app/pytest.ini]: Leave new line.

File: just_cd/deploy_app/deploy_manager.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            circleci_json=request.data,
+            branch=request.data['payload']['branch'],
+            commiter=request.data['payload']['author_name'],
+            circleci_url=request.data['payload']['build_url'],
+            vcs_revision=request.data['payload']['vcs_revision'])
+    build_info_obj.save()
+    return build_info_obj
+
+def clone_or_pull_repo(request):
+    print('in clone_or_pull_repo')
```
Comment (kracekumar) [inline just_cd/deploy_app/deploy_manager.py]: Don't use print statement. Use proper logging. http://stackoverflow.com/questions/15727420/using-python-logging-in-multiple-modules

File: just_cd/deploy_app/deploy_manager.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        git_pull(request.data['payload']['reponame'],
+                 request.data['payload']['branch'],
+                 request.data['payload']['vcs_revision'])
+    else:
+        git_clone(request.data['payload']['vcs_url'])
+
+def deploy_build(request, config_dict, allowed_branches):
+    print('in deploy_build')
+    for branch in allowed_branches:
+            if (request.data['payload']['branch'] == branch):
```
Comment (kracekumar) [inline just_cd/deploy_app/deploy_manager.py]: Spacing is out of place.

File: just_cd/deploy_app/deploy_manager.py [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+import yaml
+from django.shortcuts import get_object_or_404
+import os
+from django.conf import settings
+
+def process_status(process_output, process_status,
+                   ip, build_info_obj):
+    is_success = False
+    deployment_status = False
+    if (process_status == 0):
```
Comment (kracekumar) [inline just_cd/deploy_app/deploy_manager.py:17]: No need to brackets in Tamil.

**Pull request merged**